pkg/virt-handler/cgroup/cgroup_v1_manager.go

```diff
@@ -109,3 +113,92 @@ func getCurrentlyDefinedRules(runcManager runc_cgroups.Manager) ([]*devices.Rule
 func (v *v1Manager) GetCpuSet() (string, error) {
 	return getCpuSetPath(v, "cpuset.cpus")
 }
+
+func rw_filecontents(freadpath string, fwritepath string) (err error) {
+	rfile, err := os.Open(freadpath)
+	if err != nil {
+		return fmt.Errorf("Open failed: %s (%v)", freadpath, err)
+	}
+	defer rfile.Close()
+
+	wfile, err := os.OpenFile(fwritepath, os.O_RDWR, 0755)
+	if err != nil {
+		return fmt.Errorf("OpenFile failed: %s (%v)", fwritepath, err)
+	}
+	defer wfile.Close()
+
+	data := make([]byte, 20000)
+	count, err := rfile.Read(data)
+	if err != nil {
+		return fmt.Errorf("OpenFile failed: %s (%v)", fwritepath, err)
+	}
+
+	_, err = wfile.Write(data[:count])
+	if err != nil {
+		return fmt.Errorf("rw_filecontents: Write failed: %s", fwritepath)
+	}
```

Comment:
maybe better to use `io.Copy` here https://pkg.go.dev/io#Copy ?

Reply:
Vasiliy,

The code above looks very straightforward and simple to me.
Can't see much advantage in using the Reader/Writer interfaces.

There are some typos in the error messages though, will fix them.